services/order.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+from services.user import get_user_by_username
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> None:
+    with transaction.atomic():
+        user = get_user_by_username(username)
+        new_order = Order.objects.create(user=user)
+
+        if date:
+            new_order.created_at = date
+            new_order.save()
+
+        for ticket in tickets:
+            Ticket.objects.create(
+                movie_session_id=ticket["movie_session"],
+                order=new_order,
+                row=ticket["row"],
+                seat=ticket["seat"],
+            )
+
+
+def get_orders(username: str = None) -> QuerySet:
+    queryset = Order.objects.all().order_by("-id")
```

Comment:
You don't have to do `.order_by("-id")`. Probably, GitHub tests should pass without it.

Reply:
Pytest don't pass without that:
https://imgur.com/a/Q0WoaIn
Maybe I'm doing something else wrong